### PR TITLE
Implement Firmware page

### PR DESCRIPTION
**Pull request by nabilpandiyalayil**
- Navigation: Operations > Firmware

 - Before running, add the following env variables to .env.local file:
   - `VITE_APP_SERVER_OFF_REQUIRED=true`
   - `VITE_APP_TFTP_SERVER=false`

 - Jira: https://jsw.ibm.com/browse/PFEBMC-2389

### CODE REVIEW

File: src/views/Operations/Firmware/FirmwareAlertServerPower.vue [hunk 1]
```diff
@@ -30,22 +30,31 @@
...
-    },
-  },
-  computed: {
-    isOperationInProgress() {
-      return this.$store.getters['controls/isOperationInProgress'];
-    },
+const controlStore = ControlStore();
+
+defineProps({
+  isServerOff: {
```
Comment (Nikhil-Ashoka) [inline src/views/Operations/Firmware/FirmwareAlertServerPower.vue:41]: Looks like the warning message is not being displayed when the system is Powered On

File: src/views/Operations/Firmware/FirmwareFormUpdate.vue [hunk 2]
```diff
@@ -1,327 +1,337 @@
...
-          </b-form-group>
+            </BFormInvalidFeedback>
+          </BFormGroup>
         </template>
-        <b-btn
+        <BButton
           data-test-id="firmware-button-startUpdate"
           type="submit"
           variant="primary"
           :disabled="isPageDisabled"
```
Comment (Nikhil-Ashoka) [inline src/views/Operations/Firmware/FirmwareFormUpdate.vue:72]: The buttons are enabled when the system is powered On. Should be greyed out.

**Review by Nikhil-Ashoka — CHANGES_REQUESTED** [review]

File: src/views/Operations/Firmware/FirmwareAlertServerPower.vue (hunk 1 above)
Comment (nabilpandiyalayil) [reply src/views/Operations/Firmware/FirmwareAlertServerPower.vue:41]: We need to add `VITE_APP_SERVER_OFF_REQUIRED=true` to env variable inorder to display.

File: src/views/Operations/Firmware/FirmwareFormUpdate.vue (hunk 2 above)
Comment (nabilpandiyalayil) [reply src/views/Operations/Firmware/FirmwareFormUpdate.vue:72]: Same comment as in above.

**Review by Nikhil-Ashoka — APPROVED** [review]

**Pull request merged**